sale_order_move_menu/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Sale Order Move Menu",
+    "summary": """
+        Adds new tree view for sale moves
```

Comment:
Better something like this:
```suggestion
        Adds new menu for sale related moves
```